modules/covid_vaccine/app/jobs/covid_vaccine/expanded_registration_email_job.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+require 'va_notify/service'
+
+module CovidVaccine
+  class ExpandedRegistrationEmailJob
+    include Sidekiq::Worker
+    include SentryLogging
+    sidekiq_options expires_in: 1.day, retry: 2
+
+    STATSD_ERROR_NAME = 'worker.covid_vaccine_expanded_registration_email.error'
+    STATSD_SUCCESS_NAME = 'worker.covid_vaccine_expanded_registration_email.success'
+
+    def perform(record_id)
+      submission = CovidVaccine::V0::ExpandedRegistrationSubmission.find(record_id)
+      return if submission.email_confirmation_id.present?
+
+      notify_client = VaNotify::Service.new(Settings.vanotify.services.va_gov.api_key)
+      template_id = Settings.vanotify.services.va_gov.template_id.covid_vaccine_registration
+      email_address = submission.raw_form_data[:email]
+
+      notify_response = notify_client.send_email(email_address: email_address, template_id: template_id,
+                                                 personalisation: { 'date' => formatted_date(submission.created_at),
+                                                                    'confirmation_id' => submission.submission_uuid },
+                                                 reference: submission.submission_uuid)
+      handle_success(submission, notify_response)
+    rescue => e
+      handle_errors(e, submission&.submission_uuid)
```

Comment:
Actually going to just change this to record_id

Reply:
yeah might be easier for debugging; it's also going to be more consistent with what ActiveRecord::NotFound will include based on my testing.